pkg/scheduler/objects/node.go

```diff
@@ -312,6 +313,8 @@ func (sn *Node) RemoveAllocation(allocationKey string) *Allocation {
 		delete(sn.allocations, allocationKey)
 		sn.allocatedResource.SubFrom(alloc.GetAllocatedResource())
 		sn.availableResource.AddTo(alloc.GetAllocatedResource())
+		sn.allocatedResource.Prune()
+		sn.availableResource.Prune()
```

Comment:
The alloc should not have zeros in it. The only prune that is needed is when we do the `SubFrom`

Reply:
Addressed in latest PR, thanks!